index.bs

```diff
@@ -3242,6 +3287,43 @@ The sample code for generating and registering a new key follows:
       });
 </pre>
 
+## Registration with Platform Authenticator ## {#sample-registration-with-platform-authenticator}
+
+This is flow for when the [=[RP]=] is only interested in creating a public key credential with 
+[=platform authenticators=].
+
+1. The user visits example.com. At this point, the user likely already logged in. 
+
+1. The [=[RP]=] script runs the code snippet below. 
+
+1. The client asks the user if s/he is interested in registering a credenital with platform authenticators. 
+
+1. If the user indicates interest, proceed as described below. Otherwise, stop asking the user for interest. 
+
+1. The authenticator shows appropriate UI and direct user to create a credential. Once the new credential is created, 
```

Comment:
at this point, this scenario ought to have steps similar to, or the same as, [11.1 Registration](https://w3c.github.io/webauthn/#sample-registration) step 5 and onwards it seems.